packages/cli/src/commands/hydrogen/deploy.ts

```diff
@@ -357,10 +354,6 @@ export async function runDeploy(
     );
   }
 
-  if (isCI && envBranch) {
```

Comment:
This was added in https://github.com/Shopify/hydrogen/pull/2281, with the note:

> Future task
`env` flag will require more effort as it needs to be available on our dependant services
the current fix only patches it for the `envBranch` flag

Do we still want this feature? Is that future work tracked anywhere?